cypress/support/commands/gtl.js

```diff
@@ -8,15 +8,48 @@ Cypress.Commands.add('gtl_no_record', () => {
   return cy.get('#miq-gtl-view > div.no-record').should('be.visible');
 });
 
-Cypress.Commands.add('gtl', () => {
-  cy.get('#miq-gtl-view').then($gtlTile => {
-    if ($gtlTile.find('miq-tile-view').length > 0) {
-      return cy.get('div[pf-card-view] > .card-view-pf > .card');
-    }
-    return cy.get('#miq-gtl-view > miq-data-table > div > table');
+Cypress.Commands.add('gtlGetTable', () => {
+  return cy.get('#miq-gtl-view > .miq-data-table > .miq-data-table > .bx--data-table-content > table');
+});
+
+// columns: Array of ints of columns to read. Example: [1,2,3] will return all row data from columns 1,2 and 3.
```

Comment:
Once we've worked out the README text, these comments should be updated to match (same with the other method below)